src/scrabble.js

```diff
@@ -1,5 +1,57 @@
-function scrabble() {
+const values = require('./letter-values.js') // fetch the values table
 
```

Comment:
I'd maybe add more empty space between some lines for greater readability but love the comments, helps to easily follow what's happening